src/UpdatesLog.php

```diff
@@ -288,34 +236,83 @@ public function statusesGet(): array {
       if ($status < 0) {
         $status = '???';
       }
-      else if (empty($map[$status])) {
+      elseif (empty($map[$status])) {
         $status = '???';
       }
       else {
         $status = $map[$status];
       }
-      $statuses[$key] = $status;
+      $statuses[$key] = [
+        'status' => $status,
+        'version_used' => $data['existing_version'],
+      ];
     }
 
     return $statuses;
   }
 
   /**
-   * Check if need to run in diff mode.
+   * Generates "Statistics" of module states and verions.
    *
-   * @return bool
-   *   True if diff mode, false otherwise.
+   * @param array $statuses
+   *
+   * @return array
    */
-  public function isDiffMode(): bool {
-    return (bool) \Drupal::config('updates_log')->get('diff');
+  public function generateStatistics(array $statuses): array {
+    $statistics = [
+      "updates_log" => "2.0",
+      "last_check_epoch" => $this->lastUpdated,
+      "last_check_human" => gmdate('Y-m-d\Th:i:sZT', $this->lastUpdated),
+      "last_check_ago" => time() - $this->lastUpdated,
+      "summary" => [
+        "CURRENT" => 0,
+        "NOT_CURRENT" => 0,
+        "OUTDATED" => 0,
```

Comment:
"OUTDATED" and "NOT_CURRENT" the same thing?
or accidentally forgot "NOT_CURRENT"?

Reply:
Seems That OUTDATED somehow went in there instead of NOT_CURRENT
is fixed now